mission/src/components/ItemList/Item.vue

```diff
@@ -1,63 +1,81 @@
 <template>
-  <div class="item-list-item">
-    <img data-test="item-image" src="@/assets/images/product.svg" />
-    <div class="item-info">
-      <div class="item-discount" v-if="isDiscounted()">
-        <div class="dicount-rate" data-test="discount-rate">{{ discountRate }}</div>
-        <div class="dicount-price" data-test="discount-price">
-          {{ priceStringWithComma(price) }}
+  <div class="item-list-item w3-container">
+    <div class="w3-panel">
+      <router-link :to="{ name:'Item', params: {id: product_no}}">
```

Comment:
작성해주신 코드가 정석이기도 하지만, 이런 경우에는 편의상 아래와 같이 간단하게 구현해도 무방할 것 같습니다!
`name: 'Item'` 부분은 코드를 직접 작성하지 않은 사람 입장에서 어떤 의미인지 해석하기에 어려울 수 있을 것 같기 때문입니다 😁

```suggestion
      <router-link :to="`/item/${product_no}`">
```